lib/timodule.js

```diff
@@ -179,13 +179,13 @@ function detect(paramsOrSearchPaths, logger, callback, bypassCache) {
  * @param {Array<Object>|Object} [params.modules] - An array of modules to search for.
  * @param {Array<String>|String} [params.platforms] - An array of platform names (if the platform has more than one name) or a string of comma-separated platform names.
  * @param {Array<String>|String} [params.deployType] - An array of deploy types or a string of comma-separated deploy types to filter by.
- * @param {String} [params.sdkVersion] - The version of the Titanium SDK to be used for the minimum SDK check.
+ * @param {Object} [params.tiManifest] - The Titanium SDK manifest data.
  * @param {Array<String>} [params.searchPaths] - An array of paths to search for Titanium modules.
  * @param {Object} [params.logger] - A logger instance.
  * @param {Function} [params.callback] - A function to call when done.
  * @param {Boolean} [params.bypassCache=false] - When true, re-detects all modules.
  */
-function find(modulesOrParams, platforms, deployType, sdkVersion, searchPaths, logger, callback, bypassCache) {
+function find(modulesOrParams, platforms, deployType, tiManifest, searchPaths, logger, callback, bypassCache) {
```

Comment:
Just to be clear, technically you're passing in both an `sdkVersion` and a `moduleAPIVersion`, but due to backward compatibility you pass in via an object `tiManifest`?